official/utils/logging/logger.py

```diff
@@ -72,7 +72,7 @@ def log_estimator_evaluation_result(self, eval_results):
       if key != tf.GraphKeys.GLOBAL_STEP:
         self.log_metric(key, eval_results[key], global_step=global_step)
 
-  def log_metric(self, name, value, unit=None, global_step=None, extras=None):
+  def log_metric(self, name, value, unit=None, global_step=None, extras={}):
```

Comment:
Mutable types should not be default args.

Reply:
Done.